lxd/storage/backend_lxd.go

```diff
@@ -6300,9 +6280,37 @@ func (b *lxdBackend) UpdateCustomVolume(projectName string, volName string, newD
 		delete(newConfig, "volatile.idmap.next")
 	}
 
+	var instances []instance.Instance
```

Comment:
Can we move this new section up above the call to `b.UpdateVolume()` so we've done all the validation checks first before doing on-disk volume mutation at the driver level?